phpmd.xml

```diff
@@ -18,10 +18,10 @@
 
     <rule ref="rulesets/naming.xml/ConstantNamingConventions" />
 
-    <rule ref="vendor/mi-schi/phpmd-extension/rulesets/cleancode.xml/DataStructureMethods" />
-    <rule ref="vendor/mi-schi/phpmd-extension/rulesets/cleancode.xml/SwitchStatement" />
+    <rule ref="../../../../../mi-schi/phpmd-extension/rulesets/cleancode.xml/DataStructureMethods" />
```

Comment:
> instead of moving it, we could rename it phpmd.xml.dist and add phpmd.xml to .gitignore, just like with phpunit.xml.

Totally not get that part... we don't have phpmd dep in composer.json file anymore with this PR.
With `phpunit.xml.dist` cs `phpunit.xml` is the thing that config is automatically pick via phpunit and non-dist config is user-version, while dist is repo version. why would one create other `phpmd.xml` file? phpmd cannot automatically match used config file, user needs to chose one